app/scripts/app.js

```diff
@@ -12,6 +12,12 @@
             controller: 'timerController as timer',
             templateUrl: '/templates/timer.html'
         });
+
+        $stateProvider
```

Comment:
Since you're attaching the controller using `ng-controller` in the view and not using the more advanced ui-router features, we don't need to have this state defined.